src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -247,6 +264,88 @@ class editIndex extends viewModelBase {
             });
     }
 
+    private fetchIndexHistory() {
+        const db = this.activeDatabase();
+        
+        return new getIndexHistoryCommand(db, this.editedIndex().name() || this.originalIndexName)
+            .execute()
+            .done((indexHistory) => {
+                const indexHistoryInfo = indexHistory.History.map(x => {
+                    return { CreatedAt: x.CreatedAt, Definition: x.Definition, Source: x.Source, CreatedAtFormatted: generalUtils.formatUtcDateAsLocal(x.CreatedAt) };
+                });
+                
+                this.indexHistory(indexHistoryInfo);
+            });
+    }
+    
+    static formatTime(input: string) {
+        const dateToFormat = moment.utc(input);
+        return dateToFormat.format(editIndex.indexHistoryCreatedAtFormat) + "Z";
```

Comment:
if you don't format above - you don't need this method.

Reply:
method removed.